trimble-oss/modus-web-components
# gitignore stencil generated files

## Description

Removal of stencil-generated files from the repo and .gitignoring them.  This change will make future pull requests less polluted with autogenerated files.

References #2339

## Type of change


## How Has This Been Tested?

1. `npm run build`
2. Delete autogenerated directories
3. `npm run build` again
4. Verify that files are present in the `stencil-generated` directory and absent under unstaged files (you can run `git status` to verify this)

## Checklist
